master/getting-started/bare-metal/policy/forwarded.md

```diff
@@ -6,23 +6,34 @@ canonical_url: 'https://docs.projectcalico.org/v3.1/getting-started/bare-metal/p
 If `applyOnForward` is `false`, the host endpoint policy applies to traffic to/from
  local processes only. 
 
-If `applyOnForward` is `true`, the host endpoint policy applies to forwarded traffic:
-- Incoming traffic from a host endpoint to a local workload (container/pod/VM).
-- Outgoing traffic from a local workload to a host endpoint.
-- Traffic from one host endpoint that is forwarded to another host endpoint.
+If `applyOnForward` is `true`, the host endpoint policy also applies to forwarded traffic:
+- Traffic that comes in via a host endpoint and forwarded to a local workload (container/pod/VM).
+- Traffic from a local workload forwarded outbound via a host endpoint.
+- Traffic that comes in via host endpoint and forwarded out another host endpoint.
```

Comment:
via host endpoint -> via a host endpoint
and forwarded out -> and is forwarded via